pidcat.py

```diff
@@ -152,6 +152,8 @@ def allocate_color(tag):
 PID_DEATH = re.compile(r'^Process ([a-zA-Z0-9._:]+) \(pid (\d+)\) has died.?$')
 LOG_LINE  = re.compile(r'^([A-Z])/(.+?)\( *(\d+)\): (.*?)$')
 BUG_LINE  = re.compile(r'.*nativeGetEnabledTags.*')
+BUG_LINE2  = re.compile(r'.*glUtilsParamSize.*')
+BUG_LINE3  = re.compile(r'.*glSizeof.*')
```

Comment:
You can collapse these to a single regex `'.*(glUtilsParamSize|glSizeof|nativeGetEnabledTags).*'`.